src/components/base/Modal/ModalContent.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+
+import React, { PureComponent } from 'react'
+
+class ModalContent extends PureComponent<{
+  children: any,
+  onIsScrollableChange: boolean => void,
+}> {
+  componentDidMount() {
+    this.showHideGradient()
+    if (this._outer) {
+      const ro = new ResizeObserver(this.showHideGradient)
+      ro.observe(this._outer)
+    }
+  }
+
+  _outer = null
+
+  showHideGradient = () => {
+    if (!this._outer) return
+    const { onIsScrollableChange } = this.props
+    const isScrollable = this._outer.scrollHeight > this._outer.clientHeight
+    onIsScrollableChange(isScrollable)
+  }
+
+  render() {
+    const { children } = this.props
+
+    return (
+      <div style={CONTENT_STYLE} ref={n => (this._outer = n)}>
```

Comment:
could make this a method, or could also use React.createRef()

Reply:
absolutely :+1: